Makefile

```diff
@@ -17,3 +17,15 @@ testall: testlayouts
 # install python dependencies
 pythondeps:
 	pip install --upgrade pip && pip install  -e .
+
+# create petdafec docker image with tag (petdeface:X.X.X) from toml file by using cat and grep to 
+# extract the project name from the pyproject.toml file
+
+
+USE_LOCAL_FREESURFER ?= False
```

Comment:
perhaps a var for `DOCKER_TAG` could be helpful? Using the default from below?

e.g.
```
DOCKER_TAG ?= $(shell cat pyproject.toml | grep name | cut -d '"' -f 2):$(shell cat pyproject.toml | grep version | head -n 1 | cut -d '"' -f 2)
```

Reply:
That way you can override the tag to cut releases